src/docs/community/attributions.mdx

```diff
@@ -14,3 +14,13 @@ Attributions for assets used on this site under [CC BY](https://creativecommons.
 ## Earth Day Theme
 
 -   App Bar icons by Tezar Tantular from [the Noun Project](https://thenounproject.com/kerismaker/collection/mother-earth-day-glyph/).
+
+## Independence Day Theme
+
```

Comment:
What about the landing page banner? Where did that come from?

Reply:
Well... about that. I couldn't find the original source. It's used on various company/news/church sites over the last 5 years. I didn't see credits for the image anywhere doing a reverse image search. We can use something else if you're worried about it.

Reply:
Let's just pick one so we can CYA